types/storybook__addon-storyshots/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for @storybook/addon-storyshots 4.0
+// Type definitions for @storybook/addon-storyshots 5.1
```

Comment:
Can you [move this to an old version](https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package) instead?

Reply:
@jessetrinity Thank you for your review. But `imageSnapshot` was already removed in v4.0.0 this package. Should I still do it?

detail: https://github.com/storybookjs/storybook/releases/tag/v4.0.0-alpha.10
> `Extract imageSnapshots to a separate package #3742`

Reply:
Ah. Well if the types for 4.0 are complete and won't need to be updated and there are not breaking changes it should be fine.

Reply:
ok, please approve if there are no other problems 👍